code/modules/projectiles/guns/rifles.dm

```diff
@@ -1330,54 +1330,84 @@
 	damage_falloff_mult = 0
 	scatter = SCATTER_AMOUNT_TIER_8
 
-
 /obj/item/weapon/gun/rifle/l42a/training
 	current_mag = /obj/item/ammo_magazine/rifle/l42a/rubber
 
 //-------------------------------------------------------
 //-------------------------------------------------------
-//Basira-Armstrong rifle (Used by the CLF)
+//ABR-40 hunting rifle
 
-/obj/item/weapon/gun/rifle/hunting
-	name = "\improper Basira-Armstrong rifle"
-	desc = "Named after its eccentric designers, the Basira-Armstrong is a civilian semi-automatic rifle frequently found in the outer colonies. Despite its legally-mandated limited magazine capacity, its light weight and legendary accuracy makes it popular among hunters and competitive shooters."
-	icon_state = "hunting"
-	item_state = "hunting"
-	reload_sound = 'sound/weapons/handling/l42_reload.ogg'
-	unload_sound = 'sound/weapons/handling/l42_unload.ogg'
-	fire_sound = 'sound/weapons/gun_carbine.ogg'
-	current_mag = /obj/item/ammo_magazine/rifle/hunting
+// Civilian version of the L42A, used for hunting, and also by undersupplied paramilitary groups.
+
+/obj/item/weapon/gun/rifle/l42a/abr40
+	name = "\improper ABR-40 hunting rifle"
+	desc = "The ABR-40 was created after the striking popularity of the L42 battle rifle as a hunting rifle for civilians. Sporting faux wooden furniture and a legally-mandated 12 round magazine, it's still highly accurate and deadly, a favored pick of experienced hunters and retired Marines. However, it's very limited in attachment selection, only being able to fit rail attachments, and the differences in design from the L42 force an awkward pose when attempting to hold it one-handed. Removing the stock is not recommended."
```

Comment:
too long, put most of this into a lore desc

Reply:
yea